### ISSUE TITLE
Link to Git install

### ISSUE THREAD

**Issue by slynnf** [data-lessons/library-python-intro#24]
Setup assumes Git is installed. Perhaps link to Git install from that page.

Also explain how to "open a terminal or Git Bash" in tutorial (as is explained in video tutorial). Lesson assumes users know how to do this, but this might not be the case for all library employees.

**Comment by richyvk** [comment]
Maybe it's better to say for Windows start Spyder via the Anaconda Navigator (https://docs.continuum.io/anaconda/navigator/) does that come with the Anaconda install on Windows? Seems overkill to install git bash just to start Spyder.

**Comment by katrinleinweber** [comment]
Agreed. Can IMHO be closed, also because of https://github.com/LibraryCarpentry/lc-python-intro/issues/27.
